### DISCUSSION TITLE
Images are rotated when training

### DISCUSSION THREAD

**Discussion by zhaodadu111** [Project-MONAI/MONAI#4928] [Q&A]
hello, 
 I'm now trying to train a segmentation model for ultrasound images with dims (3,256,256). When I'm training the model, the input images are rotated 90 degree to the left. how can i rotate it back? Should i use  Orientation()? 

Thank you,
Zhaoda

**Comment by KumoLiu** [comment]
Hi @zhaodadu111, do you mean that you have checked the input and you noticed that his orientation had altered during training? In theory, training does not cause the data to change orientation. If you find your input images' orientation indeed changed, you can rotate your image using `Rotate90`. Or you can provide more information like your pre-processing process.

**Comment by wyli** [comment]
maybe you run into this issue https://github.com/Project-MONAI/MONAI/issues/4862, you can also try `Transpose`
```
>>> from monai.transforms import Transpose
>>> x = torch.zeros((3, 100, 101))
>>> Transpose((0, 2, 1))(x).shape
(3, 101, 100)
```
